src/cli/java/org/commcare/util/screen/SyncScreen.java

```diff
@@ -34,6 +24,9 @@ public class SyncScreen extends Screen {
     private PrintStream printStream;
     private boolean syncSuccessful;
 
+    public SyncScreen() {
```

Comment:
is this needed ?

Reply:
I plan to use this in Formplayer instead of passing in nulls.

I would also like to refactor this class into a base class that can have a CLI extension and a Formplayer extension since most of the functionality here is for the CLI. Having one class makes it hard to know what is used where.